SQF-VM/Entry.c

```diff
@@ -186,6 +186,12 @@ void register_commmands(PVM vm)
 	create_cmd(vm, "params", 'b', CMD_PARAMS, 4, "<ANY> params <ARRAY>", "position (\"B_Soldier_F\" createVehicle [10, 20, 30]) params [\"\", \"\", \"_z\"]; _z" "[1, nil, 2] params [\"_var1\", [\"_var2\", 23], \"_var3\"]; _var2 //23#" "private _paramsres = [1, 2] params [\"_var1\", \"_var2\", [\"_var3\", true, [true]]]; [_var3, _paramsres] //[true, false]#" "private _paramsres = [1, 2, 3] params [\"_var1\", \"_var2\", [\"_var3\", true, [true]]]; [_var3, _paramsres] //[true, false]#" "private _paramsres = [1, \"ok\", [1, 2, 3]] params [[\"_var1\", 0, [0]], [\"_var2\", "", [""]], [\"_var3\", [0,0,0], [[], objNull, 0], [2,3]]]; _paramsres //true#", "Parses input argument into array of private variables. When used without argument, as shown in main syntax, internal variable _this, which is usually available inside functions and event handlers, is used as argument.In addition to simple parsing directly into variables, input can be tested in case it is undefined, of the wrong type or of the wrong size(if array) and substituted if necessary with default values.");
 	create_cmd(vm, "createUnit", 'b', CMD_CREATEUNIT, 4, "<GROUP> createUnit <ARRAY", "", "Create unit of a class that's defined in CfgVehicles. The Group parameter MUST be an existing group or the unit won't be created.");
 	create_cmd(vm, "spawn", 'b', CMD_SPAWN, 4, "<ANY> spawn <CODE>", "", "Adds given code to the scheduler. Exactly when the code will be executed is unknown, it depends on how busy is the engine and how filled up is the scheduler. Therefore spawn does not wait for the supplied code to finish, instead, spawn returns a Script handle to the scheduler task. scriptDone command can be used to check the code completion. Additional arguments are passed to the code in local variable _this. Since Arma 3 v1.55 the script handle also exists inside the code in _thisScript variable.");
+	create_cmd(vm, "resize", 'b', CMD_RESIZE, 4, "", "", "");
```

Comment:
missing usage, examples and description (all of them)

Reply:
yeah I know... I don't want to copy the stuff from the BIKI and start formatting them in order to put them in there....

Reply:
there is no need for formatting, the others are neither formatted in any way